hvantk/commands/hgc_cli.py

```
@@ -0,0 +1,374 @@
+    ctx.obj['log_level'] = log_level
+
+    # Set up logging for HGC operations
+    setup_logging_for_hgc(log_level)
+    logger.info(f"Starting HGC command with log level: {log_level}")
+
+
+@hgc_group.command(name="gvcf-combine")
+@click.option('--gvcf-dir', '-g', help='Directory containing GVCF files')
+@click.option('--vds-paths', '-v', multiple=True, help='VDS paths to combine with GVCFs')
+@click.option('--output', '-o', required=True, help='Output VDS path')
+@click.option('--temp-dir', '--tmp', default='/tmp/hgc', help='Temporary directory for intermediate files')
```

Ruff (0.14.1)

Probable insecure usage of temporary file or directory: "/tmp/hgc"

(S108)